lib/errors.js

```diff
@@ -18,9 +18,10 @@ export default function errorObjectToProps(errorObject) {
   }
 
   // For all other errors, return an object to get rendered as an error page.
+  // serialize the error object to a string
   return {
     props: {
-      serverSideError: errorObject,
+      serverSideError: JSON.stringify(errorObject),
```

Comment:
Interesting. I wonder why this needs converting to a string when we return other objects without converting them to strings? I’ll need to look at that for igvf-ui at some point.